src/main/java/com/couchbase/lite/JavaContext.java

```diff
@@ -47,10 +51,7 @@ public SQLiteStorageEngineFactory getSQLiteStorageEngineFactory() {
 
     public File getRootDirectory() {
         String rootDirectoryPath = System.getProperty("user.dir");
-        File rootDirectory = new File(rootDirectoryPath);
-        rootDirectory = new File(rootDirectory, "data/data/com.couchbase.lite.test/files");
-
-        return rootDirectory;
+        return new File(rootDirectoryPath, "data/data/com.couchbase.lite.test/files");
```

Comment:
Can you provide example path?
Is it `<USER HOME DIR>/data/data/com.couchbase.lite.test/files` ?
Is it expected path?